### DISCUSSION TITLE
Deployment Fail: SyntaxError: Unexpected token '.'

### DISCUSSION THREAD

**Discussion by insivika** [vercel/next.js#19664] [Help]
I'm attempting to deploy my next.js app to vercel but keep running into issues using optional chaining. I downgraded node from 14 to 12.13.0 but the support person has told me that next should still support this js feature. 

![unnamed (1)](https://user-images.githubusercontent.com/33808429/100548710-12778900-3223-11eb-93cb-b64829030959.png)

**Comment by jamesmosier** [comment]
Hi @insivika. Do you have a custom `.babelrc` file? If so, could you share it?

Next supports optional chaining in both client and server code.

**Reply by insivika** [reply]
I actually don't have a custom `.babelrc` file set up 😕

**Reply by jamesmosier** [reply]
what version of Next are you using? and is there any more info you could provide?

**Reply by insivika** [reply]
here's my package.json. The project was initially built and deployed to heroku on node 14.12.0 but I declared 12.20.0 in the package.json file in order to make it work with Vercel. I've also already tried upgrading the next version from 9.5.6-canary.18 to 10.0.3. I've removed the `.bablerc` file after realizing that the `@babel/plugin-proposal-optional-chaining` plugin didnt do the trick.
```
{
  "name": "felix-homes",
  "version": "0.1.0",
  "private": true,
  "engines": {
    "node": "12.20.0"
  },
  "scripts": {
    "build": "next build",
    "start": "concurrently \"node api/server.js\" \"node scripts/scheduledSiteMapGenerator\"",
    "dev": "nodemon api/server.js"
  },
  "dependencies": {
    "@material-ui/core": "^4.11.0",
    "@material-ui/icons": "^4.9.1",
    "@material-ui/lab": "^4.0.0-alpha.56",
    "@webdeb/next-styles": "^1.1.0",
    "axios": "^0.19.2",
    "body-parser": "^1.19.0",
    "concurrently": "^5.3.0",
    "connect-timeout": "^1.9.0",
    "cors": "^2.8.5",
    "cron": "^1.8.2",
    "express": "^4.17.1",
    "formik": "^2.1.5",
    "globby": "^11.0.1",
    "heroku-ssl-redirect": "^0.1.1",
    "lodash": "^4.17.20",
    "mailgun.js": "^2.0.1",
    "moment": "^2.29.1",
    "mongoose": "^5.10.3",
    "mongoose-geojson-schema": "^2.1.5",
    "mongoose-paginate-v2": "^1.3.11",
    "next": "9.5.6-canary.18",
    "nextjs-progressbar": "0.0.7",
    "query-string": "^6.13.5",
    "rc-slider": "^9.3.1",
    "react": "16.13.1",
    "react-dom": "16.13.1",
    "react-in-viewport": "^1.0.0-alpha.14",
    "react-lazy-load-image-component": "^1.5.0",
    "react-map-gl": "^5.2.7",
    "react-number-format": "^4.4.1",
    "react-phone-number-input": "^3.0.25",
    "react-slick": "^0.27.4",
    "sass": "^1.28.0",
    "yup": "^0.29.3"
  },
  "devDependencies": {
    "@babel/core": "^7.12.9",
    "@babel/plugin-proposal-optional-chaining": "^7.12.7",
    "nodemon": "^2.0.4"
  },
  "cacheDirectories": [
    ".next/cache/"
  ]
}
```

**Reply by jamesmosier** [reply]
Unfortunately I'm not sure. I was able to run an application with the exact same setup and optional chaining worked as expected.

**Comment by ijjk** [comment]
Hi, it looks like this might be related to https://github.com/vercel/next.js/issues/17273 can you please upgrade to Next.js `v9.5.4-canary.23` or the latest version `v10.0.3` and give it a try!

**Reply by insivika** [reply]
Upgraded to `10.03` and it still wont deploy. My instals must be cached someplace, if so is there a way to purge the cache? 
package.lock: 
<img width="845" alt="Screen Shot 2020-12-01 at 2 28 28 PM" src="https://user-images.githubusercontent.com/33808429/100804397-a727e580-33e1-11eb-9347-67f6d7164afb.png">

Error Message:
![Screen Shot 2020-12-01 at 2 22 47 PM](https://user-images.githubusercontent.com/33808429/100804378-9d05e700-33e1-11eb-858f-f75df4c85176.png)
